docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -10,13 +10,13 @@ imageAlt: "Picture of the plant watering robot"
 images: ["/tutorials/plant-watering-pi/preview.png"]
 authors: ["Sierra Guequierre"]
 languages: ["python"]
-viamresources: ["board", "motor", "sensor"]
+viamresources: ["board", "motor", "sensor", "module"]
```

Comment:
This is interesting and a good update - I need to make some changes so that shows up as an option to select on the tutorials page - just fyi

Reply:
thank you!